CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [0.29.0](https://github.com/awslabs/cdk8s/compare/v0.28.0...v0.29.0) (2020-09-27)
+
+### Features
+
+* **lib:** Upgrade constructs ([#324](https://github.com/awslabs/cdk8s/issues/324)) ([070b600](https://github.com/awslabs/cdk8s/commit/070b60093bcf626deb75e8ae0f192e24797d39d8))
```

Comment:
Added this entry to avoid an empty changelog...also retitled the [PR](https://github.com/awslabs/cdk8s/issues/324).